parser/parser.go

```diff
@@ -26,9 +26,8 @@
 
```

Comment:
These and the few other unrelated changes were done by my IDE (GoLand) automatically during a merge from upstream but they seem legit to me, so I checked them in.